haystack/nodes/file_converter/azure.py

```diff
@@ -51,11 +52,13 @@ def __init__(self,
         :param save_json: Whether to save the output of the Form Recognizer to a JSON file.
         :param preceding_context_len: Number of lines before a table to extract as preceding context (will be returned as part of meta data).
         :param following_context_len: Number of lines after a table to extract as subsequent context (will be returned as part of meta data).
+        :param merge_multiple_column_headers: Whether to merge multiple column header rows to a single row.
```

Comment:
Can you explain this a bit more or give an example? I am not sure if everyone understands what a "multiple column header row" is